src/stylesheets/utilities/settings/rules/border-radius.scss

```diff
@@ -23,7 +23,8 @@ $border-radius: (
     modifiers: null,
     values: (
       0: '0',
-      pill: ('10em', isReadable),
+      pill: ('9999em', isReadable),
+      circle: ('50%', isReadable),
```

Comment:
Why is it 50% and not 100%?

Reply:
They are effectively the same, since a roundness of 50% is all you need to get to the midpoint of the square. (I have checked this!)